backend/src/pong/game_invite/entities/game-invite.entity.ts

```diff
@@ -0,0 +1,16 @@
+import { Check, Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity('game-invites')
```

Comment:
minor detail but for the constancy I thought entity names in snake case?

(copy pasta from slack)
case:
Class names = PascalCase
File names = kebab-case  (Doet nest ook bij het generaten van resources)
Column names + Table names =  snake_case (postgres bewaard alles in lower case) (edited)

Reply:
Yes!